invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/RemovalReasonsSelect.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * // This file is part of Invenio-App-Rdm
+ * // Copyright (C) 2023 CERN.
+ * //
+ * // Invenio-App-Rdm is free software; you can redistribute it and/or modify it
+ * // under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Dropdown, http, withCancel } from "react-invenio-forms";
+import { Dropdown as SUIDropdown } from "semantic-ui-react";
+
+export default class RemovalReasonsSelect extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      options: undefined,
+      loading: false,
+      defaultOpt: undefined,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchOptions();
+  }
+
+  componentWillUnmount() {
+    this.cancellableAction && this.cancellableAction.cancel();
+  }
+
+  fetchOptions = async () => {
+    const { setFieldValue } = this.props;
+    this.setState({ loading: true });
+    const url = "/api/vocabularies/removalreasons";
+    this.cancellableAction = withCancel(
+      http.get(url, {
+        headers: {
+          Accept: "application/vnd.inveniordm.v1+json",
```

Comment:
@zzacharo what about the JSON headers that you were mentioning Zenodo/InvenioRDM? Is this correct?